docs/framework/translations.md

```diff
@@ -186,9 +197,10 @@ msgstr ""
 
 5. Add your translations ins the newly created .po file. Run the app and use the
```

Comment:
Add your translations in the newly created .po file. Run the app and use the store switcher to navigate to your new storeview.